ga4gh/datamodel/variants.py

```diff
@@ -966,16 +966,20 @@ def getAnalysis(self):
             for contentKey, value in content:
                 key = "{0}.{1}".format(prefix, value.name)
                 if key not in analysis.info:
-                    analysis.info[key] = []
-                    analysis.info[key].append(value.description)
+                    analysis.info[str(key)] = []
+                if value.description is not None:
+                    analysis.info[str(key)].append(str(value.description))
         analysis.recordCreateTime = datetime.datetime.now().strftime(
             "%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
Didn't we do this isoformat stuff somewhere else already? I thought there was a ``self._createTime`` etc value somewhere; we shouldn't need to do this again here anyway.